src/wellArchitected/README.md

```diff
@@ -27,12 +27,14 @@ const query: GetPartnerDataQuery = {
         checksAgg: {
             checks: {
                 data: {
-                    count: true,
```

Comment:
Warning: Breaking change, the delete of count: true can make some problem if some projects use it.